Engine/source/console/simSet.cpp

```diff
@@ -1029,6 +1129,50 @@ DefineConsoleMethod( SimSet, getFullCount, S32, (), , "() Get the number of dire
    return object->sizeRecursive();
 }
 
+static S32 getHiddenCountRecursive(SimSet* parent)
+{
+   S32 sum = 0;
+   for(SimSet::iterator itr = parent->begin(); itr != parent->end(); itr++)
+   {
+      if((*itr)->isHidden())
+         sum++;
+      SimSet* child = dynamic_cast<SimSet*>(*itr);
+      if(child && child->size() > 0)
+         sum += getHiddenCountRecursive(child);
+   }
+   return sum;
+}
+
+DefineEngineMethod( SimSet, getHiddenCountRecursive, S32, (),,
+   "Get the number of hidden objects contained in the set and all child sets.\n"
+   "@return The number of hidden objects contained in the set and all child sets." )
+{
+   return getHiddenCountRecursive(object);
```

Comment:
Probably nitpicking here, but what possible practical use could this function have?